src/account/ProfileSubmenu.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { SettingsEvents } from 'src/analytics/Events'
+import AppAnalytics from 'src/analytics/AppAnalytics'
+import { navigate } from 'src/navigator/NavigationService'
+import { SettingsItemTextValue, SettingsExpandedItem } from 'src/components/SettingsItem'
+import User from 'src/icons/User'
+import { useSelector } from 'src/redux/hooks'
+import { phoneNumberVerifiedSelector } from 'src/app/selectors'
+import Phone from 'src/icons/Phone'
+import RevokePhoneNumber from 'src/RevokePhoneNumber'
+import { BottomSheetRefType } from 'src/components/BottomSheet'
+import CustomHeader from 'src/components/header/CustomHeader'
+import variables from 'src/styles/variables'
+import BackButton from 'src/components/BackButton'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.ProfileSubmenu>
+
+export default function ProfileSubmenu(props: Props) {
```

Comment:
This whole component is mostly a port of the same existing functionality on the current Settings screen (which will be removed once the new designs are complete)